packages/framework/src/Model/Transport/TransportFacade.php

```diff
@@ -122,22 +121,22 @@ public function getVisibleByDomainId($domainId, $visiblePaymentsOnDomain)
 
     /**
      * @param \Shopsys\FrameworkBundle\Model\Transport\Transport $transport
-     * @param \Shopsys\FrameworkBundle\Component\Money\Money[] $pricesIndexedByDomainId
+     * @param \Shopsys\FrameworkBundle\Model\Transport\TransportPricesWithVatData[] $pricesIndexedByDomainId
      */
     protected function updateTransportPrices(Transport $transport, array $pricesIndexedByDomainId): void
```

Comment:
I was a little confused by the naming... I expected something different from `$pricesIndexedByDomainId`  than a type of `TransportPricesWithVatData[]` (I expected `array<int, Price>`

Reply:
I refactored the naming after our discussion, however, I had to apply the fixups right away to save you from solving the conflicts next week :smile: